src/otx/core/utils/config.py

```diff
@@ -66,3 +67,29 @@ def mmconfig_dict_to_dict(obj: MMConfig | list[MMConfig]) -> list | dict:
         return {k: mmconfig_dict_to_dict(v) for k, v in obj.to_dict().items()}
 
     return obj
+
+
+def inplace_num_classes(
```

Comment:
It seems this function is duplicated with https://github.com/openvinotoolkit/training_extensions/blob/0e8bd95b41e899c01ec2fee2a6258d24f6f08598/src/otx/core/utils/build.py#L82
Can we choose one of them?

Reply:
Please deprecate it after merging it.
https://github.com/openvinotoolkit/training_extensions/blob/0e8bd95b41e899c01ec2fee2a6258d24f6f08598/src/otx/core/utils/build.py#L89
This should be `isinstance(value, ListConfig)`.